src/main/java/org/embulk/output/DatabricksOutputPlugin.java

```diff
@@ -52,6 +53,20 @@ public interface DatabricksPluginTask extends PluginTask {
     @Config("delete_stage_on_error")
     @ConfigDefault("false")
     public boolean getDeleteStageOnError();
+
+    @Config("user_agent")
+    @ConfigDefault("{}")
+    public Optional<UserAgentEntry> getUserAgent();
```

Comment:
```suggestion
    public UserAgentEntry getUserAgentEntry();
```

- If there is a default value, then `Optional` is not needed.
- If there is no special reason, it should be the same as the option name.

Reply:
fixed in [this commit](https://github.com/trocco-io/embulk-output-databricks/pull/4/commits/0e4ed29571b0345865c38361ed630f50a3e3d09f).